### PR TITLE
Make keys as symbols

### PR DESCRIPTION
**Pull request by bogdan-getstream**
# Submit a pull request

## CLA

- [x] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required).
- [x] The code changes follow best practices
- [x] Code changes are tested (add some information if not applicable)

## Description of the pull request
Solves #11 
Added HashWithIndifferentAccess for json parsing

### CODE REVIEW

File: lib/stream-chat/indifferent_hash.rb [hunk 1]
```diff
@@ -0,0 +1,192 @@
+# Source from https://github.com/sinatra/sinatra/blob/master/lib/sinatra/indifferent_hash.rb
```
Comment (keyneston) [inline lib/stream-chat/indifferent_hash.rb:1]: For a copy of this size we should be maintaining license file. Can this go in a subfolder along with the LICENSE from the original project?

Comment (bogdan-getstream) [reply lib/stream-chat/indifferent_hash.rb:1]: Is it not enough to provide link? the code have MIT license

Comment (keyneston) [reply lib/stream-chat/indifferent_hash.rb:1]: No, we have to distribute a copy of the license and the copyright info with the code. The alternative is to copy the license file that is [currently distributed with the project](https://github.com/sinatra/sinatra/blob/master/LICENSE) and put it in a comment at the top of this file.

**Comment by ferhatelmas** [comment]
rubocop fixes this.

**Pull request closed without merging**